### ISSUE TITLE
Assertion failed: `p.system || p.skeleton.has_value (), file C:\build2-build\build2-toolchain-0.17.0\bpkg\bpkg\pkg-build.cxx, line 7475`

### ISSUE THREAD

**Issue by helmesjo** [build2/build2#414]
Tested and reproducible on Windows.
```
$ bdep --version
bdep 0.17.0
libbpkg 0.17.0
libbutl 0.17.0
```
[assert-failed.zip](https://github.com/user-attachments/files/16522889/assert-failed.zip)
```
$ b libcore/
warning: configured src_root ..\qme\pkgs\libcore\ does not match forwarded libcore\
Assertion failed: p.system || p.skeleton.has_value (), file C:\build2-build\build2-toolchain-0.17.0\bpkg\bpkg\pkg-build.cxx, line 7475
error: process bpkg terminated abnormally: aborted
..\qme-build\x86_64-windows-msvc-debug\build\bootstrap\pre-bdep-sync.build:10:9: error: process C:\build2\bin\bdep.exe exited with code 1
  info: command line: C:\build2\bin\bdep.exe sync --hook=1 --verbose 1 --config C:\Users\fho\git\qme-build\x86_64-windows-msvc-debug
```

**Comment by karen-arutyunov** [comment]
Thanks, for the report. Can't reproduce though.

Could you provide the exact steps for how to reproduce the issue? Also, could you run the crashing `b libcore/` command with the `--verbose 6` option and provide the output?

**Comment by helmesjo** [comment]
Unfortunately it seems that I also can't reproduce it anymore. Most have been related to the state of the configuration (which I've since cleaned and re-initiated).
I'll keep an eye out for it happening again.

**Comment by boris-kolpackov** [comment]
@helmesjo This inability to reproduce does happen quite regularly in such complex cases (understandably), so I think going forward it makes sense to always attache the `--verbose 6` output. This will give us at least something to investigate. Thanks!

**Comment by helmesjo** [comment]
Yep, agreed! In this case I repeated the reproduce-steps in a random directory but I guess it probably still contains an absolute path to the configuration, hence why I was still able to reproduce it then.
